indigo_app/urls.py

```diff
@@ -102,6 +102,8 @@
     re_path(r'^works(?P<frbr_uri>/\S+?)/edit/$', works.EditWorkView.as_view(), name='work_edit'),
     re_path(r'^works(?P<frbr_uri>/\S+?)/approve$', works.ApproveWorkView.as_view(), name='work_approve'),
     re_path(r'^works(?P<frbr_uri>/\S+?)/unapprove$', works.UnapproveWorkView.as_view(), name='work_unapprove'),
+    re_path(r'^works(?P<frbr_uri>/\S+?)/edit-offcanvas/$', works.EditWorkOffCanvasView.as_view(), name='work_edit_offcanvas'),
```

Comment:
Please drop the trailing `/` - I'd like to be more consistent and not use a slash in general